src/dotnet/CarbonAware.WebApi/README.md

```diff
@@ -1,3 +1,5 @@
+The Carbon Aware SDK provides api to get the carbon intensity for a location at a given time period. In order to use the following API, the environment has to be setup with the right configuration. Instructions on setting up the environment could be found here - https://github.com/microsoft/carbon-aware-sdk/blob/dev/GettingStarted.md
+
 # Carbon Aware REST API
```

Comment:
IMO, it would be redundant to have this information on each call. I think each API call should ideally only document what the call does with information about the input, output payload.

Reply:
I agree with George.  Swagger will list this information and make it easy to submit sample calls.

We should also tell them how to get to the swagger endpoint, once they have the WebAPI running.

Reply:
Sounds good.